hangups/http_utils.py

```diff
@@ -46,6 +46,9 @@ def fetch(method, url, params=None, headers=None, cookies=None, data=None,
         else:
             error_msg = None
             break
+        finally:
+            if 'res' in locals():
+                res.close()
```

Comment:
From reading [the aiohttp docs](http://aiohttp.readthedocs.io/en/stable/client_reference.html#aiohttp.ClientResponse.read), I'm not sure that closing the response is the right thing to do. If `read` raises an exception, it will close the connection for us, and otherwise, it releases the connection, which allows keep-alive.

The bug in `_longpoll_request` might be where it checks the response status code and raises an exception manually without closing the response.